clients/web/src/views/widgets/HierarchyWidget.js

```diff
@@ -681,6 +707,21 @@ girder.views.HierarchyWidget = girder.View.extend({
         }).on('g:saved', function (folder) {
             // need to do anything?
         }, this);
+    },
+
+    /**
+     * Select (highlight) an item in the list.
+     * @param item An ItemModel instance representing the item to select.
+     */
+    selectItem: function (item) {
```

Comment:
What calls `selectItem`?   Is it something that other javascript would call?  If so, can you add a test doing so?  Similarly with `getSelectedItem`.